.storybook/preview.js

```diff
@@ -67,7 +67,7 @@ addDecorator(
         path: "/", // defaults to `/`
         asPath: "/", // defaults to `/`
         query: {}, // defaults to `{}`
-        push() {}, // defaults to using addon actions integration, can override any method in the router
+        // push() {}, // defaults to using addon actions integration, can override any method in the router
```

Comment:
This was preventing `router.push` messages from being logged in Storybook's `Actions` tab.
I thought I would leave it commented out as an example (though I'm not sure about that syntax, and VSCode highlighted it with a warning).